.github/CONTRIBUTION.md

```diff
@@ -0,0 +1,53 @@
+## Developing
+
+1. Install with `yarn`
+1. Transpile code: `yarn build` (or `yarn build:watch`)
+1. Link to sense-client (locally on your computer):
```

Comment:
Not sure we want this step here, you need access to sense-client. @niekvanstaveren

Reply:
No we should not have this in the open source repo. Probably only mention yarn start here.

Reply:
done